modules/common/reddit.js

```diff
@@ -5,7 +5,7 @@ var require_install = require('require-install'),
 
 exports.reddit = function(thing, numberOfQueries, callback) {
     reddit.r(thing).limit(numberOfQueries, function(err, req, res){
-        if (err || req !== null || req !== undefined) {
+        if (!err || req !== null || req !== undefined) {
```

Comment:
Should those be ands? If there is no error or the request is null or the request is undefined? That or if there is an error?